Sources/Xcodeproj/XcodeProjectModelSerialization.swift

```diff
@@ -64,7 +64,9 @@ extension Xcode.Project: PropertyListSerializable {
             dict["productRefGroup"] = .identifier(serializer.id(of: productGroup))
         }
         dict["projectDirPath"] = .string(projectDir)
-        dict["targets"] = .array(targets.map({ target in
+        // Ensure that targets are output in a sorted order.
+        let sortedTargets = targets.sorted { $0.name.localizedStandardCompare($1.name) == .orderedAscending }
```

Comment:
We try to avoid `Foundation` where possible. Can you make this `targets.sorted(by: { $0.name < $1.name })`?

Reply:
Not a problem. Would you prefer I avoid trailing closure syntax as well?

Reply:
We don't have an official style guide yet, but we try to use the well known patterns in the Swift community. This one comes under the "[The Rule of Kevin](http://ericasadun.com/2015/11/17/a-handful-of-swift-style-rules-swiftlang/)".

Reply:
Makes sense, I tend to use trailing closures regardless of that distinction, but I'm happy to follow the project standards. I'll update this now 👍